scripts/new-version-pr-generator.js

```diff
@@ -39,28 +39,34 @@ const polarisBotToken = secrets.github['shopify-polaris'];
 const shopifyPolarisBotGitOverride = `GIT_COMMITTER_NAME='${polarisBotName}' GIT_COMMITTER_EMAIL='${polarisBotEmail}'`;
 
 function pullRequestTemplate(repository, version) {
-  return `
-## Version ${version} of @shopify/polaris just got published!
+  let prodAssetChecklist = '';
 
-See what’s new: https://github.com/Shopify/polaris-react/releases/tag/${version}
+  if (repository === 'web') {
+    prodAssetChecklist = `
+      ### How to [tophat](https://development.shopify.io/engineering/developing_at_Shopify/write-code/tophatting)
 
----
+      <!-- Tophatting instructions, and/ or what you want reviewers to concentrate on. -->
 
-### How to [tophat](https://development.shopify.io/engineering/developing_at_Shopify/write-code/tophatting)
+      ⚠️ This PR must be tophatted with [production assets](https://github.com/Shopify/web/blob/master/documentation/guides/assets.md) because it updates a key dependency (Polaris React).
```

Comment:
Markdown should be unindented, otherwise indented lines will be rendered as code blocks.

Reply:
TIL!